redis/commands/get.cc

```diff
@@ -48,8 +48,7 @@ future<redis_message> get::execute(service::storage_proxy& proxy, redis::redis_o
         if (result->has_result()) {
             return redis_message::make_strings_result(std::move(result->result()));
         }
-        // FIXME: we should returns nil string if key does not exist
-        return redis_message::err();
+        return redis_message::nil();
```

Comment:
Does this fix issue https://github.com/scylladb/scylla/issues/5317 ?
If so, please add the "Fixes" line.
If not, can we add a new test for what you're fixing here?

Reply:
Yes, I will update the commitlog.

I saw @fastio already posted another PR to fix this issue, I will remove this patch from this PR.